website/docs/components/form/file-input/partials/code/component-api.md

```diff
@@ -0,0 +1,67 @@
+## Component API
+
+The File Input component has two different variants with their own APIs:
+
+- `Form::FileInput::Base`—the base component: the `<input type="file">` control
+- `Form::FileInput::Field`—the field parent component: the `<input type="file">` control, with label, helper text, and error messaging (in a wrapping container)
+
+### Form::FileInput::Base
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+    <br/><br/>
+    The attributes will be applied to the `<input type="file">` element. This means you can use [all the standard HTML attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file) of the `<input type="file">` element and all the usual Ember techniques for event handling, validation, etc.
+    <br/><br/>
+    Examples of HTML attributes: `id`, `name`, `value`, `placeholder`, `disabled`, `readonly`, `required`. See [the whole list of HTML attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attributes). Examples of Ember modifiers: `\{{on "input" [do something]}}`, `\{{on "change" [do something]}}`, `\{{on "blur" [do something]}}`.
```

Comment:
[blocker] We should probably update this list, some of these attributes I suspect don't apply to a `input[type=file]`